src/screens/HomeScreen.tsx

```diff
@@ -130,37 +135,82 @@ const HomeScreen = ({navigation}: any) => {
   }
 
   return (
-    <ScrollView
-      style={styles.container}
-      bounces={false}
-      contentContainerStyle={styles.scroll_view_container}>
+    <ScrollView style={styles.container} bounces={false}>
       <StatusBar hidden />
       <View style={styles.search_bar_input}>
         <SearchBar search_functionality={search_functionality} />
       </View>
       <CategoryHeader>Now Playing</CategoryHeader>
+      <FlatList
+        horizontal
+        bounces={false}
+        data={now_playing_movies_list}
+        keyExtractor={item => item.id}
+        contentContainerStyle={[styles.flatlist_container]}
+        snapToInterval={width * 0.7 + SPACING.space_4}
+        decelerationRate={0}
```

Comment:
Prefer to provide the value of `decelerationRate` as string shortcuts out of either `normal`(which is by default) or `fast`. This match the underlying iOS settings for `UIScrollViewDecelerationRateNormal` and `UIScrollViewDecelerationRateFast` respectively.

Refer: https://reactnative.dev/docs/scrollview#decelerationrate